examples/express-openapi-validator/app.js

```diff
@@ -0,0 +1,77 @@
+import { App } from '@tinyhttp/app'
+
+import fs from 'fs'
+import { OpenApiValidator } from 'express-openapi-validate'
+import jsYaml from 'js-yaml'
+import bodyParser from 'body-parser'
```

Comment:
not that much required but could you please use [milliparsec](https://github.com/talentlessguy/milliparsec) instead of body-parser?